config/manager/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+resources:
+- manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: secret-sync-controller
```

Comment:
```suggestion
  newName: secrets-store-sync-controller
```

Reply:
This comment applies to other places as well - let's use `secrets-store-sync-controller` where applicable